mailbox/store/src/test/java/org/apache/james/mailbox/store/StoreMailboxMessageResultIteratorTest.java

```diff
@@ -76,7 +76,12 @@ public int content() {
         public TestMessageMapper(MessageRange messageRange) {
             this.messageRange = messageRange;
         }
-        
+
+        @Override
+        public Iterator<MessageUid> getUids(Mailbox mailbox) throws MailboxException {

```

Comment:
Because it allows implementations to not load the entire data structure at once, and I don't have Streams (not java-8)